.eslintignore

```diff
@@ -1,2 +1,3 @@
 bundle.js
-main.bundle.js
\ No newline at end of file
+main.bundle.js
+App.spec.js
```

Comment:
I see you remembered Eliza's bundle.js warning. Do we need to exclude tests?

Reply:
@rickboot I actually have to ask Eliza about this. I will check it out and comment it here. Thanks!